wiegand/api.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+from RPi.GPIO import *
+import time
+import json
+
+class Wiegand26:
+    #Reads pin numbers for input pins from JSON
+    #Facility number is also set from JSON
+    def __init__(self):
+        config = {}
+        with open('config.json', 'r') as config_file:
+            config = json.load(config_file)
+        self.data0 = config["data0"]
+        self.data1 = config["data1"]
+        self.facility = config["facility"]
+
+    #Sets data pins to IN
+    def setup(self):
+        setwarnings(False)
+        setmode(BOARD)
+        setup(self.data0, IN)
+        setup(self.data1, IN)
+
+    # When Card returns wrong Facility 
```

Comment:
you might need to do some research on how to structure these error exceptions. putting them here doesn't look so nice.

also, i'm seeing you `pass` them. why don't they return anything meaningful, like error code or message,...?

Reply:
I asked andrew and he said that it was fine to have a class within a class as long as it was an exception

Reply:
i agree. it's fine to leave it here. it's just not a good practice.

Reply:
should i make an exceptions.py instead?

Reply:
yes, that's a good idea. i would suggest to look for how other people do it. this is a nice [example](https://github.com/kennethreitz/requests/blob/master/requests/exceptions.py). note how they structure their project too.